marimbabot_speech/src/speech_extraction.py

```diff
@@ -0,0 +1,170 @@
+import os
+from nvidia import cudnn
+os.environ['LD_LIBRARY_PATH'] = f'{os.path.dirname(cudnn.__file__)}/lib:$LD_LIBRARY_PATH'
+# since I haven't specified the version of TensorFlow, it seems to install the newest version,
+# so according to official instruction, the environment needs to be configured.
+from precise_runner import PreciseEngine, PreciseRunner, ReadWriteStream
+from audio_common_msgs.msg import AudioDataStamped, AudioData
+from std_msgs.msg import Float32
+import rospy
+import webrtcvad
+from playsound import playsound
+
+VAD_ON = False
```

Comment:
Why are these globals and not class fields?

Reply:
TODO: put the global variable into constructor.